Formula/zeek.rb

```diff
@@ -47,5 +47,6 @@ def install
E1hYyBPUyBYIDEwLjE2LCBidWlsZCAyMEQ2NCAoRGFyd2luIDIwLjMuMCkABAAyAER1bXBjYXAgKFdpcmVzaGFyaykgMy40LjMgKHYzLjQuMy0wLWc2YWU2Y2QzMzVhYTkpAAAAAAAAxAAAAAEAAABkAAAAAQAAAAAACAACAAMAZW4wAAMABQBXaS1GaQAAAAkAAQAGAAAADAArAE1hYyBPUyBYIDEwLjE2LCBidWlsZCAyMEQ2NCAoRGFyd2luIDIwLjMuMCkAAAAAAGQAAAAGAAAAcAAAAAAAAAAZvwUAPM7xYk4AAABOAAAAdIPCb6a48BiYOtvGCABFAABAAABAAEAGTKzAqAUzaBXAG+C/AFA5uK2ZAAAAALAC//9hXwAAAgQFtAEDAwYBAQgKyApXJAAAAAAEAgAAAABwAAAABgAAAGQAAAAAAAAAGb8FAJWb8mJCAAAAQgAAAPAYmDrbxnSDwm+muAgARSgANAAAQAAyBlqQaBXAG8CoBTMAUOC/rbsbeTm4rZqAEv//75YAAAIEBXgBAQQCAQMDCgAAZAAAAAYAAABYAAAAAAAAABm/BQDum/JiNgAAADYAAAB0g8JvprjwGJg628YIAEUAACgAAEAAQAZMxMCoBTNoFcAb4L8AUDm4rZqtuxt6UBAQACAwAAAAAFgAAAAGAAAApAAAAAAAAAAZvwUAcpzyYoEAAACBAAAAdIPCb6a48BiYOtvGCABFAABzAABAAEAGTHnAqAUzaBXAG+C/AFA5uK2arbsbelAYEACo7AAAR0VUIC8gSFRUUC8xLjENCkhvc3Q6IGlmY29uZmlnLmNvDQpVc2VyLUFnZW50OiBjdXJsLzcuNjQuMQ0KQWNjZXB0OiAqLyoNCg0KAAAApAAAAAYAAABcAAAAAAAAABm/BQCIVPNiPAAAADwAAADwGJg628Z0g8JvprgIAEUoACi2qUAAMgaj8mgVwBvAqAUzAFDgv627G3o5uK3lUBAAQC+lAAAAAKz1ZZtcAAAABgAAAGgDAAAAAAAAGb8FAEPY9WJFAwAARQMAAPAYmDrbxnSDwm+muAgARSgDN7aqQAAyBqDiaBXAG8CoBTMAUOC/rbsbejm4reVQGABAORYAAEhUVFAvMS4xIDIwMCBPSw0KRGF0ZTogU2F0LCAwMyBBcHIgMjAyMSAyMjo1NzoxOCBHTVQNCkNvbnRlbnQtVHlwZTogdGV4dC9wbGFpbjsgY2hhcnNldD11dGYtOA0KQ29udGVudC1MZW5ndGg6IDE1DQpDb25uZWN0aW9uOiBjbG9zZQ0KU2V0LUNvb2tpZTogX19jZmR1aWQ9ZDlhYjVhYjg0M2I5NTFkNWFiNmY5ZWI0MWQ5Zjk4ZGIyMTYxNzQ5MDYzODsgZXhwaXJlcz1Nb24sIDAzLU1heS0yMSAyMjo1NzoxOCBHTVQ7IHBhdGg9LzsgZG9tYWluPS5pZmNvbmZpZy5jbzsgSHR0cE9ubHk7IFNhbWVTaXRlPUxheA0KQ0YtQ2FjaGUtU3RhdHVzOiBEWU5BTUlDDQpjZi1yZXF1ZXN0LWlkOiAwOTNiOGQ4ZmIwMDAwMDQ5NTUxZjI1MTAwMDAwMDAwMQ0KUmVwb3J0LVRvOiB7Imdyb3VwIjoiY2YtbmVsIiwiZW5kcG9pbnRzIjpbeyJ1cmwiOiJodHRwczpcL1wvYS5uZWwuY2xvdWRmbGFyZS5jb21cL3JlcG9ydD9zPWR3NTZGV3JGN1JYN2JZQ2ttcEpucTh6VkQzVHlIZDQ4MSUyRmlxSjgxQld4SWhZWTFwdkFjS0RVeTFYMW9oZlNmWGN4WkhKTWpvQlljcmxHRjN4OU54enlOMHB4YmxNWEdKeTBZNEp3JTNEJTNEIn1dLCJtYXhfYWdlIjo2MDQ4MDB9DQpORUw6IHsicmVwb3J0X3RvIjoiY2YtbmVsIiwibWF4X2FnZSI6NjA0ODAwfQ0KU2VydmVyOiBjbG91ZGZsYXJlDQpDRi1SQVk6IDYzYTVlNTJjNGJlMzQ5NTUtU1RMDQphbHQtc3ZjOiBoMy0yNz0iOjQ0MyI7IG1hPTg2NDAwLCBoMy0yOD0iOjQ0MyI7IG1hPTg2NDAwLCBoMy0yOT0iOjQ0MyI7IG1hPTg2NDAwDQoNCjE5OS44My4yMjAuMTE1CgAAAGgDAAAGAAAAXAAAAAAAAAAZvwUAStj1YjwAAAA8AAAA8BiYOtvGdIPCb6a4CABFKAAotqtAADIGo/BoFcAbwKgFMwBQ4L+tux6JObit5VARAEAslQAAAABiTwpPXAAAAAYAAABYAAAAAAAAABm/BQCd2PViNgAAADYAAAB0g8JvprjwGJg628YIAEUAACgAAEAAQAZMxMCoBTNoFcAb4L8AUDm4reWtux6JUBAP8xzjAAAAAFgAAAAGAAAAWAAAAAAAAAAZvwUAndj1YjYAAAA2AAAAdIPCb6a48BiYOtvGCABFAAAoAABAAEAGTMTAqAUzaBXAG+C/AFA5uK3lrbseilAQD/Mc4gAAAABYAAAABgAAAFgAAAAAAAAAGb8FAELZ9WI2AAAANgAAAHSDwm+muPAYmDrbxggARQAAKAAAQABABkzEwKgFM2gVwBvgvwBQObit5a27HopQERAAHNQAAAAAWAAAAAYAAABcAAAAAAAAABm/BQCQjfZiPAAAADwAAADwGJg628Z0g8JvprgIAEUoACgAAEAAMgZanGgVwBvAqAUzAFDgv627Hoo5uK3mUBAAQCyUAAAAADaoONVcAAAABQAAAGwAAAAAAAAAGb8FAEVEC2MBABwAQ291bnRlcnMgcHJvdmlkZWQgYnkgZHVtcGNhcAIACAAZvwUAw83gYgMACAAZvwUAHkQLYwQACAAOAAAAAAAAAAUACAAAAAAAAAAAAAAAAABsAAAA' | base64 -d | #{bin}/zeek -C -r - && test -s conn.log && test -s http.log")
```

Comment:
What's the expected output of the call to `zeek` here?

Reply:
Alas, there's no output. That's why I'm using the `test -s` for the presence of the non-zero-sized output files of those particular names as evidence of a successful run. Typically Zeek is run more like a server process watching live interfaces so watching its output interactively is not very common.

Regarding your question about how the `base64_input` was generated, I just did `base64 ifconfig.pcapng` on the pcap (an uncompressed copy of this binary attachment: [ifconfig.pcapng.gz](https://github.com/Homebrew/homebrew-core/files/6255002/ifconfig.pcapng.gz)). The binary pcap file could surely live external to the Formula, but I got the sense that you liked having everything in standalone single files, so I used the base64 trick to get the binary data in there.

I just pushed an additional commit that uses your suggestion to satisfy the "strict audit". I just had to add a `,0` in the `pipe_output` so the exit code would be checked for success. In that state I was able to force it to fail by mangling one of the expected filenames.

Reply:
Expecting a return code of `0` is implicit -- it's only needed if you actually expect a non-zero exit code (as the `test` block will fail otherwise).

Can we download a pcap file we can use here from somewhere? It just needs to be a stable URL. Otherwise, we might also consider using some other formula to generate it (we just need a `depends_on "formula" => :test`) if `zeek` can't do it on its own.

Reply:
@carlocab: Sure, I've just pushed another commit that pulls a pcap from a stable URL within the Zeek repo itself. I'd gone the base64 route because I've worked on other projects where the maintainers really hated the idea of intermittent network glitches when accessing URLs causing CI to fail. But this works just fine.

Reply:
Thanks for your work here. I just have one small revision to make, which I'll push to your branch.